frontend/packages/dev-console/integration-tests/support/pages/app.ts

```diff
@@ -0,0 +1,202 @@
+import { devNavigationMenu, switchPerspective } from '../constants/global';
+
+export const app = {
+  waitForLoad: (timeout: number = 30000) => {
+    cy.get('.co-m-loader', { timeout }).should('not.be.visible');
+  },
+  login: () => {
+    cy.get('body').then(($body) => {
+      if ($body.find('a[title="Log in with kube:admin"]').length) {
+        cy.get('a[title="Log in with kube:admin"]').click();
+        cy.url().should('include', 'login');
+      }
+    });
+    cy.get('#inputUsername').type(Cypress.env('username'));
+    cy.get('#inputPassword').type(Cypress.env('password'));
+    cy.get('[type="submit"]').click();
+    cy.get('[aria-label="Help menu"]').should('be.visible');
+  },
+};
+
+export const perspective = {
+  verifyPerspective: (perspectiveName: string) => {
+    cy.byLegacyTestID('perspective-switcher-toggle').should('contain.text', perspectiveName);
+  },
+
+  switchTo: (perspectiveName: switchPerspective) => {
```

Comment:
Pls see [nav](https://github.com/openshift/console/blob/52986b6fca9befc99af0581339ec004b201db13c/frontend/packages/integration-tests-cypress/views/nav.ts#L6) helper object for this functionality.

Reply:
I think we can use enums as parameters, so that it would be easier for others to use. WDYT?